lib/openingScreen.dart

```diff
@@ -0,0 +1,111 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+import 'menu.dart';
+
+class openingScreen extends StatefulWidget {
+  @override
+  State<openingScreen> createState() => _openingScreenState();
+}
+
+class _openingScreenState extends State<openingScreen> with TickerProviderStateMixin{
+  late AnimationController controller;
+  @override
+  void initState() {
+
+    controller=AnimationController(vsync: this,
+    duration: const Duration(seconds: 1),
+
+    )..addListener(() {
+      setState(() {
+
+      });
+
+    }
+
+    );
+
+    controller.repeat(reverse: true);
+    // super.initState();
+
+    Timer(Duration(seconds: 1), () {
+      Navigator.pushReplacement(
```

Comment:
This is very important.

Under the `lib/extensions/navigation.dart`, I have defined the logic for navigation.
You do not need to call Navigator.pushReplacement() all the time no longer. Instead, you can use `context.pushReplacement(Widget page)`
Example:
```dart
context.pushReplacement(const InductionAppMenu());
```
You do not need to call MaterialPageRoute anymore.